pages/checkout/redirect.tsx

```diff
@@ -0,0 +1,82 @@
+import { Progress, Typography } from "@equinor/eds-core-react"
+// import { tokens } from "@equinor/eds-tokens"
+import type { NextPage } from "next/types"
+import {
+  ReactNode,
+  useCallback,
+  useEffect,
+  useState,
+} from "react"
+import { FormattedMessage, useIntl } from "react-intl"
+import styled from "styled-components"
+
+import { CheckoutWizard } from "../../components/CheckoutWizard/CheckoutWizard"
+import { Container } from "../../components/Container"
+
+const Ingress = styled(Typography).attrs({ variant: "ingress" })`
+  margin-bottom: 0.75rem;
+`
+
+const HelpText = styled(Typography)`
+  margin-bottom: 1.5rem;
+`
+
+const CheckoutRedirectView: NextPage = () => {
+  const [progress, setProgress] = useState<number>(0)
+  const intl = useIntl()
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setProgress((p) => {
+        if (p === 100) return 100
+        return p + 2
+      })
+    }, 100)
+
+    return () => clearInterval(interval)
+  }, [])
+
+  useEffect(() => {
+    if (progress === 100) {
+      window.open("https://accessit.equinor.com", "_blank")
+    }
+  }, [progress])
+
+  const FormattedLink = useCallback((chunks: ReactNode[]) => (
+    <Typography
+      link
+      href="https://accessit.equinor.com"
+      target="_blank"
+      rel="noopener noreferrer nofollow"
+    >
+      {chunks}
+    </Typography>
+  ), [])
+
+  return (
+    <Container>
+      <CheckoutWizard>
+        <Typography variant="h4" as="h1" style={{ marginBottom: "0.25rem" }}>{intl.formatMessage({ id: "checkout.redirect.headline" })}</Typography>
```

Comment:
Inline styles instead of styled component = inconsistency? 🤔

Reply:
i know, and i hate it ☹️ but i had an issue where if i created an inherited styled component, the `as` prop would cause the component to receive browser default styles for `h1` and lose styles set by eds